devtools/src/arcs-dataflow.html

```diff
@@ -337,10 +337,7 @@
         return data.length > 100 ? data.substring(0, 100) : data;
       }
 
-      _indentPrint(thing) {
-        if (typeof thing === "string") thing = JSON.parse(thing)
-        return JSON.stringify(thing, null, 2);
-      }
+      _indentPrint(thing) {return indentPrint(thing);} // from arcs-shared
```

Comment:
This wouldn't pass lint if it wasn't in the .html file.

Reply:
Fixed.